lib/db/structs.proto

```diff
@@ -28,6 +28,7 @@ message FileInfoTruncated {
     uint32                permissions    = 4;
     int64                 modified_s     = 5;
     int32                 modified_ns    = 11;
+    uint64                modified_by    = 12;
```

Comment:
This, and in bep.proto, should be

``` 
uint64       modified_by    = 12 [(gogoproto.customtype) = "ShortID", (gogoproto.nullable) = false];
```

to get the correct type on the Go side of things.